xdsl/dialects/llvm.py

```diff
@@ -1735,56 +1734,132 @@ def __init__(self, val: Operation | SSAValue, res_type: Attribute):
         )
 
 
+class AbstractFloatArithOp(IRDLOperation, ABC):
+    T: ClassVar = VarConstraint("T", BaseAttr(AnyFloat))
```

Comment:
```suggestion
    T: ClassVar = VarConstraint("T", AnyFloatConstr)
```
I think this exists for this reason